### PR TITLE
Fix livesync help

### PR DESCRIPTION
**Pull request by rosen-vladimirov**
Fix livesync help.

### CODE REVIEW

**Comment by vchimev** [comment]
:+1:

File: docs/man_pages/project/testing/build-android.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ Command | Description
 [emulate ios](emulate-ios.html) | Builds the specified project and runs it in the native iOS Simulator.
 [emulate](emulate.html) | You must run the emulate command with a related command.
 [livesync](livesync.html) | Synchronizes the latest changes in your project to devices.
-[livesync ios](livesync-ios.html) | Synchronizes the latest changes in your project to iOS devices.
+[livesync ios](livesync-ios.html) | Synchronizes the latest changes in your project to iOS devices or the iOS simulator.
```
Comment (ikoevska) [inline docs/man_pages/project/testing/build-android.md]: iOS Simulator - we need the capital letter :)

File: docs/man_pages/project/testing/debug-android.md [hunk 2]
```diff
@@ -23,7 +23,7 @@ Debugs your project on a connected device, in a native emulator or in Genymotion
 * `--stop` - Detaches the debug tools.
 * `--get-port` - Retrieves the port on which you are debugging your application.
 * `--debug-port` - Sets a new port on which to attach the debug tools.
-* `--timeout` - Sets the number of seconds that NativeScript CLI will wait for the debugger to boot. If not set, the default timeout is 90 seconds.
+* `--timeout` - Sets the number of seconds the NativeScript CLI will wait for the debugger to boot. If not set, the default timeout is 90 seconds.
```
Comment (ikoevska) [inline docs/man_pages/project/testing/debug-android.md]: that the 

You can keep it as it is but the additional "that" might bring more clarity for non-native English speakers.

File: docs/man_pages/project/testing/build-android.md (hunk 1 above)
Comment (rosen-vladimirov) [inline docs/man_pages/project/testing/build-android.md]: ok, I'll update it everywhere

File: docs/man_pages/project/testing/livesync-ios.md [hunk 3]
```diff
@@ -3,19 +3,26 @@ livesync ios
...
+* `--emulator` - If set, synchronizes changes to the iOS Simulator. You cannot set `--device` and `--emulator` simultaneously.
 
 ### Attributes
 * `<Device ID>` is the device index or identifier as listed by `$ tns device`
 
 <% if(isHtml) { %> 
+### Command Limitations
+
+* You cannot set `--device` and `--emulator` simultaneously.
+* You can livesync only to iOS simulator running 8.x runtime.
```
Comment (ikoevska) [inline docs/man_pages/project/testing/livesync-ios.md]: * For the iOS Simulator, you can synchronize changes only to iOS 8.x devices.

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,6 +1,28 @@
...
+
+### New
+* [Implemented #470](https://github.com/NativeScript/nativescript-cli/issues/470): Add timeout option to debug command.
+* [Implemented #682](https://github.com/NativeScript/nativescript-cli/issues/682): Set `--debug-brk` as default option of `tns debug` command.
+* [Implemented #706](https://github.com/NativeScript/nativescript-cli/issues/706): LiveSync to iOS Simulator devices running iOS 8.x.
+
+### Fixed
+
+* [Fixed #470](https://github.com/NativeScript/nativescript-cli/issues/470): Command failed due to space in file path.
+* [Fixed #632](https://github.com/NativeScript/nativescript-cli/issues/632): Android debugger does not work on some OSX systems.
```
Comment (ikoevska) [inline CHANGELOG.md]: OS X

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,6 +1,28 @@
...
+* [Fixed #652](https://github.com/NativeScript/nativescript-cli/issues/652): Debug command for iOS does not load inspector from the added ios-runtime.
+* [Fixed #663](https://github.com/NativeScript/nativescript-cli/issues/663): Verify that Java is at least 1.7.
+* [Fixed #671](https://github.com/NativeScript/nativescript-cli/issues/671): iOS debugger doesn't work for projects created with framework version lower than 1.2.0.
```
Comment (ikoevska) [inline CHANGELOG.md]: framework versions earlier than

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,6 +1,28 @@
...
+* [Fixed #679](https://github.com/NativeScript/nativescript-cli/issues/679): Library add command adds the framework with its full path into xcode project.
```
Comment (ikoevska) [inline CHANGELOG.md]: `tns library add`

Actually apply this comment to all places where you mention X command.

Comment (ikoevska) [inline CHANGELOG.md]: Xcode

Comment (ikoevska) [inline CHANGELOG.md]: Ok, let's make this 

`tns library add ios` adds the framework with its full path in the Xcode project.

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,6 +1,28 @@
...
+* [Fixed #695](https://github.com/NativeScript/nativescript-cli/issues/695): Exceptions tracking is not correctly set in user-settings.
+* [Fixed #699](https://github.com/NativeScript/nativescript-cli/issues/699): Unable to update android platform when npm cache is in inconsistent state.
```
Comment (ikoevska) [inline CHANGELOG.md]: Android

in an inconsistent state

File: CHANGELOG.md [hunk 8]
```diff
@@ -1,6 +1,28 @@
...
+* [Fixed #722](https://github.com/NativeScript/nativescript-cli/issues/722): Debug command builds and installs the app twice.
+* [Fixed #742](https://github.com/NativeScript/nativescript-cli/issues/742): LiveSync doesn't work on some android devices.
```
Comment (ikoevska) [inline CHANGELOG.md]: Android

File: CHANGELOG.md [hunk 9]
```diff
@@ -1,6 +1,28 @@
...
+* [Fixed #747](https://github.com/NativeScript/nativescript-cli/issues/747): New files are not respected from livesync command.
```
Comment (ikoevska) [inline CHANGELOG.md]: from -> by

File: docs/man_pages/project/testing/build-ios.md [hunk 10]
```diff
@@ -34,7 +34,7 @@ Command | Description
 [emulate ios](emulate-ios.html) | Builds the specified project and runs it in the native iOS Simulator.
 [emulate](emulate.html) | You must run the emulate command with a related command.
 [livesync](livesync.html) | Synchronizes the latest changes in your project to devices.
-[livesync ios](livesync-ios.html) | Synchronizes the latest changes in your project to iOS devices.
+[livesync ios](livesync-ios.html) | Synchronizes the latest changes in your project to iOS devices or the iOS simulator.
```
Comment (ikoevska) [inline docs/man_pages/project/testing/build-ios.md]: My capital S is missing here :P

File: docs/man_pages/project/testing/livesync-ios.md [hunk 11]
```diff
@@ -3,19 +3,26 @@ livesync ios
...
 
-Synchronizes the latest changes in your project to iOS devices or iOS simulator. 
+Synchronizes the latest changes in your project to iOS devices or the iOS Simulator.
 
 ### Options
 * `--watch` - If set, when you save changes to the project, changes are automatically synchronized to the connected device.
-* `--device` - Specifies the serial number or the index of the connected device to which you want to synchronize changes. To list all connected devices, grouped by platform, run `$ tns device`
-* `--emulator` - If set, changes are synchronized to iOS simulator.
+* `--device` - Specifies the serial number or the index of the connected device to which you want to synchronize changes. To list all connected devices, grouped by platform, run `$ tns device`. You cannot set `--device` and `--emulator` simultaneously.
+* `--emulator` - If set, synchronizes changes to the iOS Simulator. You cannot set `--device` and `--emulator` simultaneously.
```
Comment (ikoevska) [inline docs/man_pages/project/testing/livesync-ios.md]: We can drop the "If set" here.

File: docs/man_pages/project/testing/livesync-ios.md [hunk 12]
```diff
@@ -3,19 +3,26 @@ livesync ios
...
 
 ### Attributes
 * `<Device ID>` is the device index or identifier as listed by `$ tns device`
 
 <% if(isHtml) { %> 
+### Command Limitations
+
+* You cannot set `--device` and `--emulator` simultaneously.
+* For the iOS Simulator, you can synchronize changes only to iOS 8.x devices
```
Comment (ikoevska) [inline docs/man_pages/project/testing/livesync-ios.md]: .

File: CHANGELOG.md [hunk 13]
```diff
@@ -1,6 +1,28 @@
 NativeScript CLI Changelog
 ================
 
+1.2.2 (2015, August 7)
+==
+
+### New
+* [Implemented #470](https://github.com/NativeScript/nativescript-cli/issues/470): Add timeout option to debug command.
```
Comment (ikoevska) [inline CHANGELOG.md]: code formatting for `--timeout` option

File: CHANGELOG.md [hunk 14]
```diff
@@ -1,6 +1,28 @@
...
+* [Implemented #682](https://github.com/NativeScript/nativescript-cli/issues/682): Set `--debug-brk` as default option of `tns debug` command.
```
Comment (ikoevska) [inline CHANGELOG.md]: the default option of `tns debug`

And drop command 

Btw is this for tns debug or for tns debug android

Comment (rosen-vladimirov) [inline CHANGELOG.md]: it is already working for ios, we've just added it for Android

File: docs/man_pages/project/testing/build-ios.md (hunk 10 above)
Comment (rosen-vladimirov) [inline docs/man_pages/project/testing/build-ios.md]: :crying_cat_face:

File: CHANGELOG.md [hunk 15]
```diff
@@ -1,6 +1,28 @@
...
+
+### Fixed
+
+* [Fixed #470](https://github.com/NativeScript/nativescript-cli/issues/470): Command failed due to space in file path.
+* [Fixed #632](https://github.com/NativeScript/nativescript-cli/issues/632): Android debugger does not work on some OSX systems.
+* [Fixed #652](https://github.com/NativeScript/nativescript-cli/issues/652): Debug command for iOS does not load inspector from the added ios-runtime.
+* [Fixed #663](https://github.com/NativeScript/nativescript-cli/issues/663): Verify that Java is at least 1.7.
+* [Fixed #671](https://github.com/NativeScript/nativescript-cli/issues/671): iOS debugger doesn't work for projects created with framework version lower than 1.2.0.
+* [Fixed #679](https://github.com/NativeScript/nativescript-cli/issues/679): Library add command adds the framework with its full path into xcode project.
+* [Fixed #695](https://github.com/NativeScript/nativescript-cli/issues/695): Exceptions tracking is not correctly set in user-settings.
```
Comment (ikoevska) [inline CHANGELOG.md]: probably user settings

alternatively user-settings.xml

**Comment by ikoevska** [comment]
:+1:

**Pull request merged**